scss/_alert.scss

```diff
@@ -4,44 +4,64 @@
 
 .alert {
   position: relative;
-  padding: $alert-padding-y $alert-padding-x;
+  padding: from-pixels(12);
```

Comment:
This breaks multi-tenancy. There should be no hard coded values outside of _variables.scss

Reply:
Updating